ios/Capacitor/Capacitor/Plugins/Keyboard.m

```diff
@@ -42,28 +42,38 @@ @implementation CAPKeyboard
 - (void)load
 {
   [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(statusBarDidChangeFrame:) name:UIApplicationDidChangeStatusBarFrameNotification object: nil];
+    
   NSString * style = [self getConfigValue:@"style"];
   if ([style isEqualToString:@"dark"]) {
     [self setKeyboardAppearanceDark];
   }
-  
-  self.keyboardResizes = ResizeNative;
+    
   BOOL doesResize = YES;
-  if (!doesResize) {
+  if ([[self getConfigValue:@"resize"] isEqualToString:@"none"]) {
+    doesResize = NO;
     self.keyboardResizes = ResizeNone;
-    NSLog(@"CAPIonicKeyboard: no resize");
+    NSLog(@"CAPKeyboard: no resize");
+  }
+
+  if (doesResize) {
+    self.keyboardResizes = ResizeNative;
+    NSString * resizeMode = [self getConfigValue:@"resize"];
     
-  } else {
-    NSString *resizeMode = @"ionic";
     if (resizeMode) {
       if ([resizeMode isEqualToString:@"ionic"]) {
         self.keyboardResizes = ResizeIonic;
+        NSLog(@"CAPKeyboard: resize mode - ionic");
       } else if ([resizeMode isEqualToString:@"body"]) {
         self.keyboardResizes = ResizeBody;
+          NSLog(@"CAPKeyboard: resize mode - body");
       }
     }
-    // NSLog(@"CAPIonicKeyboard: resize mode %d", self.keyboardResizes);
+      
+    if (self.keyboardResize == ResizeNative) {
```

Comment:
I think It should be **keyboardResizes** instead of **keyboardResize**.  It throws an error that keyboardResize is not defined.

Reply:
Indeed. The "s" was lost, when I changed config param name. Thanks!